documentation/main.md

```diff
@@ -108,6 +115,35 @@ URI that contains both dialect and driver.
         "postgresql+psycopg://myuser:mypass@myserver:1234/mydatabase"
     )
 
+## Views
+
+The method to make views described [here](https://github.com/sqlalchemy/sqlalchemy/wiki/Views#sqlalchemy-14-20-version)
+is available in this library.
+
+This allows creating views like this:
+```
+org_type_view = view(
```

Comment:
Would it be possible to add (simplified) tables to this example?

Reply:
Good idea! I have done so